### DISCUSSION TITLE
PHY Testvectors Issue

### DISCUSSION THREAD

**Discussion by Justin-Garey** [srsran/srsRAN_Project#824] [General Help] [answered]
I was trying to test with the PHY testvectors but I'm having an issue with at least one of the binaries not being available. `tests/unittests/phy/generic_functions/transform_precoding/transform_precoder_test_data.tar.gz` is not in the latest testvectors zip file and is causing an error with cmake.

As a side note, the docs for the PHY testvectors state to use ```tar -xzf phy_testvectors.tar.gz -C /path_to_your_local_repository/srsgnb/``` but the tar file is actually a `POSIX tar archive (GNU)` so the ```-z``` should be taken out of the flags to ```tar```. I was unable to unzip until correcting that. 

I think the instructions could also be more helpful by mentioning that the directory to copy the contents to should be created before unzipping and that the contents need copied over to the actual test directory. For that I used: ```cp -R srsgnb/tests/* tests/```.

**Comment by dvdgrgrtt** [comment, marked as answer]
Hi Justin,
sorry for the trouble. Unfortunately, because of some automation issues, test vectors are updated only with major releases, so it is indeed possible that they don't match the latest version of the repo.
I'm sharing a copy of the latest vectors I have just generated - everything should work fine. I will also see to fix the format of the file.
https://drive.google.com/file/d/1q5OFOt7H-8JwR0FCKxxXgovgz0rZW3Ex/view?usp=drive_link
